packages/dashboard/src/components/csvDownloadButton/index.tsx

```diff
@@ -7,6 +7,24 @@ import { StyledSiteWiseQueryConfig } from '~/customization/widgets/types';
 import { useViewportData } from '~/components/csvDownloadButton/useViewportData';
 import { IoTSiteWiseClient } from '@aws-sdk/client-iotsitewise';
 
+const EMPTY_DATA = [
```

Comment:
What do you think about making a CSV constants file in this folder for these types of things?

Reply:
+1